conductor-client/src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -386,7 +386,17 @@
                             ID_MAP,
                             VERSION )
                     .distributionColumn( LINKING_ID );
-
+    // needed for migration
+    public static final  PostgresTableDefinition LEGACY_PERMISSIONS        =
```

Comment:
Understand why we have this, but should also make sure we have SQL to create this iron'd out (should be a simple inner join filtering by securable object types).

Reply:
Yup, I have the SQL detailed in the migration ticket itself.